src/locales/en/v1/every-error-explained-in-detail.adoc

```diff
@@ -2,16 +2,31 @@
 == Every Error Explained in Detail
 :nofooter:
 
-[[importerror]]
-=== Import Error
+[[othererrors]]
+=== Overview
+
+An error is a mistake in the code that prevents the program from running.
```

Comment:
hm... not all errors prevent programs from running.

Reply:
In this sense, we're talking about `throw Error` though. Is it possible to throw an error without the program stopping?

Maybe I can re-word this to avoid the issue.

Reply:
From the curriculum 1.4
> Sometimes programmers make mistakes that cause code to work incorrectly, or not run at all. In programming, coding faults are called errors, or bugs. The process of finding and fixing bugs is called debugging. You can use debugging strategies, using the console.